releasenotes/notes/secret_backend_command_sha256-b322140f976f02e1.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Adds support for `secret_backend_command_sha256` SHA for the `secret_backend_command` executable. If `secret_backend_command_sha256` is used,
+    the following restrictions are in place:
+    - Value specified in the `secret_backend_command` setting must be an absolute path.
+    - Permissions for the datadog.yaml config file must disallow write access by unprivileged accounts.
+    The agent will refuse to start if the actual SHA256 of 'secret_backend_command' executable is different from the one specified by `secret_backend_command_sha256`.
```

We need double backticks since `reno` produce a RST in the end for the CHANGELOG.
```suggestion
    Adds support for ``secret_backend_command_sha256`` SHA for the ``secret_backend_command`` executable. If ``secret_backend_command_sha256`` is used,
    the following restrictions are in place:
    - Value specified in the ``secret_backend_command`` setting must be an absolute path.
    - Permissions for the datadog.yaml config file must disallow write access by unprivileged accounts.
    The agent will refuse to start if the actual SHA256 of ``secret_backend_command`` executable is different from the one specified by ``secret_backend_command_sha256``.
```

---

Ah, thank you so much!